### ISSUE TITLE
Pre-trained models on MOS dataset

### ISSUE THREAD

**Issue by Runsong123** [yashbhalgat/Contrastive-Lift#2]
Dear author,

Thanks for your previous help and providing the pretrained model on Scannet. I would really appreciate it if you could further provide the pretrained model (Constrastive-Lifting and Panoptic-Lifting)  in the MOS datasets if possible since I have limited computation. 
Many thanks in advance!!

Bests,
Runsong

**Comment by vrk7** [comment]
@Runsong123 Can you let me know if you are training on a different dataset? If yes, can you help me by saying how to do that?

**Comment by Runsong123** [comment]
> @Runsong123 Can you let me know if you are training on a different dataset? If yes, can you help me by saying how to do that?

Yes, I just modify the corresponding data path to train the Contrastive-lifting on a different dataset.
